code/game/turfs/simulated/floor.dm

```diff
@@ -43,12 +42,17 @@ var/list/wood_icons = list("wood","wood-broken")
 		icon_regular_floor = "floor"
 	else
 		icon_regular_floor = icon_state
+	if(floor_tile)
+		builtin_tile = new floor_tile
+
+	spawn(5)
+		update_visuals()

```

Comment:
It's part of how we handle LINDA's turf visuals- The atmos tanks do not render correctly without calling this a bit after they spawn